test/shared_test/test_ortops_strings.cc

```diff
@@ -292,10 +283,7 @@ TEST(utils, test_string_join) {
   outputs[0].dims = {1};
   outputs[0].values_string = {"abc-zzz-efg"};
 
-  std::filesystem::path model_path = __FILE__;
```

Comment:
android NDK doesn't support __FILE__ or the file cannot be accessed in the test app?

Reply:
The model file should be accessed by relative path instead of absolute path, because the data file will be pushed to Android emulator and we can't keep the same path. So I remove `__FILE__`.